src/main/java/it/gov/pagopa/exception/ConsentException.java

```diff
@@ -0,0 +1,18 @@
+package it.gov.pagopa.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+@AllArgsConstructor
+@Getter
+@Setter
+public class ConsentException extends RuntimeException {
```

Comment:
Why an exception specific for "Consent" purpose and not a generic one (eg: ClientException)?